src/Components/StatusBar/StatusBar.js

```
@@ -0,0 +1,40 @@
+import {StatusBarMenu} from "./StatusBarMenu/StatusBarMenu";
+
+import "./StatusBar.scss";
+
+StatusBar.propTypes = {
+};
+
+/**
+ * Status bar of the viewer component.
+ * @return {JSX.Element}
+ */
+export function StatusBar ({}) {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)